### PR TITLE
Styling

### PR DESCRIPTION
**Pull request by daudi13**
## Description
This is the last project of the Bookstore website. Now that all the functionality is done, you have to add styling. You will use Zeplin as a collaboration tool.

- Styled my project using the design provided on Zeplin
### General requirements

- [x] There are no linter errors.
- [x] Followed the **Github flow**.
- [x] The documentation is professional.

### HTML/CSS requirements

- [x] Followed best practices for HTML & CSS
- [x] Followed best practices for JavaScript

### Project requirements

- [x] You will find all the details of the Bookstore design in [Zeplin](https://zeplin.io/.)
- [x] Login to Zeplin (hint: you will find the credentials in the sneak peek of the project).
- [x] Open the project Bookstore CMS.
- [x] Use the information detailed in Zeplin to style the website.
- [x] Your final design should match Zeplin's design.

---

Thank you for taking the time to review this pull request.

### CODE REVIEW

File: src/components/Navbar/Navbar.css [hunk 1]
```diff
@@ -1,31 +1,57 @@
+
 .navBar {
+  width: 100vh;
+  width: 100%;
+}
```
Comment (Dipeshtwis) [inline src/components/Navbar/Navbar.css]: - [x] You are using the same property twice here which would make confusion, please use the correct one.

File: src/components/Home/BookForm.css [hunk 2]
```diff
@@ -18,16 +18,20 @@ form > .form-container {
...
   margin-right: 2rem;
+  border: 1px solid #e8e8e8;
 }
 
 .category {
+  border: 1px solid #e8e8e8;
   padding: 0.375rem 9.938rem 0.563rem 1rem;
   margin-right: 2rem;
+  color: grey;
 }
```
Comment (Dipeshtwis) [inline src/components/Home/BookForm.css]: - [ ] I would recommend using the DRY principle here. 

```css

.input, .category {
  border: ....;
  margin-right: 2rem;
}

.input {
  ...
  ...
}

.category {
  ...
  ...
}
```

**Review by Dipeshtwis — CHANGES_REQUESTED** [review]
Hi @daudi13, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
:+1: Styling looks good to me.

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N.A.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Navbar/Navbar.css (hunk 1 above)
Comment (daudi13) [reply src/components/Navbar/Navbar.css]: Noted

File: src/components/Home/BookForm.css (hunk 2 above)
Comment (daudi13) [reply src/components/Home/BookForm.css]: Thanks for pointing that out :+1:

**Review by BenMukebo — APPROVED** [review]
### Hi @daudi13, 👋 

# STATUS: Approved  🎉 

Great work on making the changes requested by a previous reviewer :1st_place_medal:  
 Your project is complete!  There is nothing else to say other than... it's time to merge it
Congratulations! 🎉  🎉 🎉 




## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**